app/models/ancestor_ref.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AncestorRef < ActiveRecord::Base
```

Comment:
`AncestorRef` is a very specific name but IMO doesn't describe its purpose. Maybe `LatestRelease`? That could help show that it's intended to be kept up to date (for those that need to stay up to date)

Reply:
Following up, `Service` as @caitlinrhd suggested would likely work better than `LatestRelease`

Reply:
updated to be `Service` like you and Caitlin suggested below 👍